app/models/concerns/foreman/event_subscribers/observable.rb

```diff
@@ -0,0 +1,36 @@
+module Foreman
+  module EventSubscribers
```

Comment:
I suggest to pick a name in core and stick with it. Since you want to use "webooks" in your plugin and I am going to keep "foreman_hooks" name, let's change this to HookSubscriber perhaps?